src/exportable/common.ts

```diff
@@ -98,3 +98,14 @@ export function fshifyString(input: string): string {
     .replace(/\r/g, '\\r')
     .replace(/\t/g, '\\t');
 }
+
+export function splitOnPathPeriods(path: string): string[] {
+  return path.split(/\.(?![^\[]*\])/g); // match a period that isn't within square brackets
+}
```

Comment:
Is this function necessary? I think the paths we are using this function on are coming from FHIR definitions right. On SUSHI this function is needed when we are splitting a FSH path, so the path may have something like a URL within the parentheses. I don't think that could happen here. Let me know if I'm missing something here.

Reply:
No I think you're right. I had been using a regular `split('.')` and then at the last minute decided to switch to use that function to be safe. But I think you're right that these will only be FHIR definition paths, so the regular `split` should be sufficient. I'll switch back!

Reply:
Fixed in 7183f39